src/components/m-accordion-item/_template.js

```diff
@@ -0,0 +1,34 @@
+import bel from 'bel';
+import classnames from 'classnames';
+import { BaseComponent } from '../_abstract/component-types';
+
+export default ({ header, headersecondary, headercolor, classes, multiple, open = false, icon }, childrenFragment) => {
+  const headerPrimaryClasses = classnames('m-accordion-item__header__primary', {
+    [`m-accordion-item__header__primary--${headercolor}`]: headercolor,
+    'm-accordion-item__header__primary--with-secondary': headersecondary,
+  });
+  const headerClasses = classnames('m-accordion-item__header', {
+    'm-accordion-item__header__primary--with-icon': icon,
+  });
+
+  const iconToRender = bel`<axa-icon class="m-accordion-item__header__icon-wrapper" id="${icon}" classes="m-accordion-item__header__icon"></axa-icon>`;
```

Comment:
better not to style the axa-icon itself. axa-icon uses classes:

<img width="536" alt="bildschirmfoto 2018-02-22 um 10 20 39" src="https://user-images.githubusercontent.com/275235/36529944-12d41b56-17ba-11e8-9224-ebf776c7b2d3.png">